yt_dlp/extractor/daystar.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+from .common import InfoExtractor
+from ..utils import js_to_json, try_get, urljoin
+
+
+class DaystarClipIE(InfoExtractor):
+    IE_NAME = 'daystar:clip'
+    _VALID_URL = r'https?:\/\/player\.daystar\.tv\/(?P<id>\w+)($|\W)'
+    _DOMAIN = 'https://www.lightcast.com/embed/'
+    _TEST = {
+        'url': 'https://player.daystar.tv/0MTO2ITM',
+        'info_dict': {
+            'id': '0MTO2ITM',
+            'ext': 'm3u8',
+            'title': 'The Dark World of COVID Pt. 1 | Aaron Siri',
+        },
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id=video_id)
+
+        title = self._html_search_meta(['og:title', 'twitter:title'], webpage, fatal=True)
+        description = self._html_search_meta(['og:description', 'twitter:description'], webpage, fatal=True)
+
```

Comment:
inline

Reply:
Sorry, but could you explain, what does 'inline' mean?

Reply:
he means a variable is not needed here and this can be inlined in the return statement
```py
return {
    ...
    'description': self._html_search_meta(['og:description', 'twitter:description'], webpage, fatal=True),
    ...
}
```